docs/mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: SKNNR Documentation
+repo_url: https://github.com/lemma-osu/scikit-learn-knn-regression
+repo_name: lemma-osu/scikit-learn-knn-regression
+docs_dir: pages/
+
+nav:
+  - Home: index.md
+  - Installation: installation.md
+  - Usage: usage.md
+  - "API Reference":
+    - Estimators:
+      - RawKNNRegressor: api/estimators/raw.md
+      - EuclideanKNNRegressor: api/estimators/euclidean.md
+      - MahalanobisKNNRegressor: api/estimators/mahalanobis.md
+      - GNNRegressor: api/estimators/gnn.md
+      - MSNRegressor: api/estimators/msn.md
+    - Transformers:
+      - StandardScalerWithDOF: api/transformers/standardscalerwithdof.md
+      - MahalanobisTransformer: api/transformers/mahalanobis.md
+      - CCATransformer: api/transformers/cca.md
+      - CCorATransformer: api/transformers/ccora.md
+    - Datasets:
+      - Dataset: api/datasets/dataset.md
+      - "Moscow Mountain / St. Joes": api/datasets/moscow_stjoes.md
+      - "SWO Ecoplot": api/datasets/swo_ecoplot.md
+  - Contributing: contributing.md
+
+theme: 
+  name: material
+  features: 
+    - search.suggest
+    - search.highlight
+    - navigation.instant
+    - navigation.path
+    - content.code.copy
+    - content.code.annotate
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      toggle:
+        icon: material/weather-night
+        name: Dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/weather-sunny
+        name: Light mode
+
+plugins:
+  - search
+  - mkdocstrings:
+      handlers:
+        python:
+          paths: [../src]
+          options:
+            show_source: false
```

Comment:
Do you have a preference on this option? Personally I like when docs have a link to the source code, but this seemed a little too obtrusive, and I don't believe there's any config to modify.

Reply:
I totally agree that I like the presence of a link to the source code, but I'm thinking you mean the presentation of the source code (when switching `show_source` to `true`) is too obtrusive without the capability to modify it.  If I understood this correctly, I do agree that it's a bit too "present" in the docs.  

I prefer the style of `scikit-learn` itself where it takes you (in a new page) to the Github source.  That's a shame there isn't any way to modify that.

Reply:
Yes, exactly - I was also hoping for something more like the `[source]` link in the `sklearn` docs. I found mkdocstrings/mkdocstrings#525 that had a similar complaint, but no resolution. I might poke around and see what it would take to make a PR over there, but for now I guess we can stick with no source code and revisit later.